config.go

```diff
@@ -165,6 +166,8 @@ func LoadConfig(path string, isFile bool) error {
 	viper.SetDefault("derp.server.enabled", false)
 	viper.SetDefault("derp.server.stun.enabled", true)
 
+	viper.SetDefault("randomize_ip", false)
```

Comment:
in the config you have `randomize_address` either change the config example or the viper access